packages/rum-core/test/common/context.spec.js

```diff
@@ -165,6 +165,25 @@ describe('Context', () => {
     })
   })
 
+  it('should add navigation timing span context', () => {
+    const url = 'https://example.com'
+    const span = new Span(url, 'hard-navigation')
+    span.end()
+    addSpanContext(span, { url })
+
+    expect(span.context).toEqual({
+      destination: {
+        service: {
+          name: 'https://example.com',
+          resource: 'example.com:443',
+          type: 'hard-navigation'
```

Comment:
What is the reason for using a separate type here? Would it make a difference in the UI?

Reply:
I kept it similar to the `span.type`. I went over the UI code to see the difference. The UI relies on the `span.type` field for the icons and not the `span.destination.service.type`. So if we need the same treatment as for Resources, We would need to change our Navigation timing spans `type` and `subtype`. 

Icon logic - https://github.com/elastic/kibana/blob/e70fcc708e727ba6c2b0d4145b136ef3d75a42a5/x-pack/plugins/apm/public/components/app/ServiceMap/icons.ts#L94-L104

Resource spans get `globe` icon - https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/public/components/app/ServiceMap/icons/globe.svg